src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -94,6 +97,8 @@ public function load($resource, $type = null)
                     $controller = $this->parser->parse($controller);
                 } catch (\Exception $e) {
                     // unable to optimize unknown notation
+                } catch (\Throwable $e) {
+                    // unable to optimize unknown notation
```

Comment:
-1 for this one. Getting an Error in the parser should not happen when it is an unknown notation (the parser is expected to throw an Exception in such case), so it would only hide bugs